packages/client/hmi-client/src/App.vue

```diff
@@ -36,10 +63,14 @@ const dataExplorerActivated = ref(false);
 		@show-overlay="enableOverlay"
 		@hide-overlay="disableOverlay"
 	/>
-	<Header class="header" @show-data-explorer="dataExplorerActivated = true" />
+	<Header
+		class="header"
+		:projectName="project?.name"
```

Comment:
You can take out this `?` since project will always be an instance of `Project`.

```suggestion
		:projectName="project.name"
```

Though, what do you think about using `null` when no project is selected rather than `ProjectService.empty`?
That way when components need to know if a project has been selected they can just check `project === null` rather than having to know the arbitrary ID of the empty project constant.

Reply:
You are correct, I was lazy and didn't wanted to handle `null`, as `emptyProject` was already existing.